pyproject.toml

```diff
@@ -40,6 +40,9 @@ pytest-watch = "^4.2.0"
 twine = "^3.8.0"
 wheel = "^0.37.1"
 
+[tool.poetry.group.dev.dependencies]
+pytest-order = "^1.3.0"
```

Comment:
to run integration tests in order

Reply:
Is this because they're actually interdependent now?

Reply:
yeah